### PR TITLE
Save movie

### PR DESCRIPTION
**Pull request by jacobtie**

### CODE REVIEW

File: templates/create_movies_form.html [hunk 1]
```diff
@@ -5,5 +5,19 @@
 {% block body %}
 <h1 class="mb-5">Create Movie Rating</h1>
 <p class="mb-3">Create a movie rating below</p>
-<!-- TODO: Feature 2 -->
+<form action="/movies" method="post">
+  <div class="mb-3">
+    <label for="title">Title</label>
+    <input type="text" name="title" class="form-control" autocomplete="off">
```
Comment (DylanHalstead) [inline templates/create_movies_form.html:11]: These and other inputs should probably be required

File: tests/unit/test_create_movie.py [hunk 2]
```diff
@@ -1 +1,13 @@
-# TODO: Feature 2
+from src.repositories import movie_repository
+
+mrs = movie_repository.get_movie_repository()
+
+
+def test_create_movie():
+    mrs.clear()
+    movie = mrs.create_movie('Test Create Movie', 'Test Create Director', 3)
```
Comment (DylanHalstead) [inline tests/unit/test_create_movie.py:8]: I love 'Test Create Movie', favorite film

**Review by DylanHalstead — APPROVED** [review]
lgtm, check comments in code

File: templates/create_movies_form.html (hunk 1 above)
Comment (jacobtie) [reply templates/create_movies_form.html:11]: Yeah maybe

File: tests/unit/test_create_movie.py (hunk 2 above)
Comment (jacobtie) [reply tests/unit/test_create_movie.py:8]: Same

**Review by DylanHalstead — APPROVED** [review]
Swag

**Pull request merged**